### DISCUSSION TITLE
Ability to add additional modles to Ollama

### DISCUSSION THREAD

**Discussion by dgoryeo** [SubtitleEdit/subtitleedit#8308] [Ideas]
I noticed that when I add a new model to Ollama, SE does not detect them. It would be useful to have a feature (if not already in place) that allows user add-on models. 

Example, I wanted to use this model : https://ollama.com/sunapi386/llama-3-lexi-uncensored 

Thanks

**Comment by Gameon666** [comment]
additional ollama models that are not listed in the "model" drop space, just simply type it in the blank space to activate a model of your choice. see attached pic 
![2024-07-25_180645](https://github.com/user-attachments/assets/3588c46d-5276-45d4-9c38-b683f28b6f3a)
![2024-07-25_180718](https://github.com/user-attachments/assets/5ccb6198-2f93-4e2f-bfdd-3272d8428c15)
in my case, i am using the "llama3.1:8b-instruct-q6_k" model and it is pretty fast, though translation still needs fine-tune. 

PS: in your case you need to pull the desired model from ollama in CMD then "activate" it in SE

**Comment by niksedk** [comment]
They should be in latest version of SE... and you can right-click in the model combobox and choose "Refresh local models"

**Comment by niksedk** [comment]
`llama3.1` is super slow on my computer :(
